src/main/java/com/exactpro/th2/fix/client/FixMessage.java

```diff
@@ -7,29 +7,30 @@
 
 public class FixMessage extends Message {
 
-    public static final int DEFAULT_TIME_TO_LIVE = 30;
+    public static final int DEFAULT_TIME_TO_LIVE_SECONDS = 30;
+    private static final ExpiringMap<String, EventID> MESSAGE_PARENT_EVENT_IDS = new ExpiringMap<>(DEFAULT_TIME_TO_LIVE_SECONDS);
 
     protected final FixHeader header;
     protected final FixTrailer trailer;
     protected EventID parentEventID;
-    public static ExpiringMap<String, EventID> parentEventIDs = new ExpiringMap<>(DEFAULT_TIME_TO_LIVE);
+    protected Message message;
```

Comment:
`FixMessage` class must act a decorator (e.g. proxy calls of methods we're not interested in to `mesage`) for this `message` which adds ability store parent-event id. Right it's somewhat of a holder